contracts/dao/staking/interfaces/IStakingEvents.sol

```diff
@@ -5,12 +5,13 @@
 pragma solidity 0.8.13;
 
 interface IStakingEvents {
-    event Staked(address account, uint256 indexed streamShares, uint256 indexed nVoteToken, uint256 indexed lockId);
+    event Staked(address account, uint256 amount,uint256 indexed streamShares, uint256 indexed nVoteToken, uint256 indexed lockId, uint256 end);
```

Comment:
why streamShares and nVoteToken are indexed and account not?

Reply:
Removed indexed for steamShares and nVoteToken. Added indexed for account.